jenkins-pipelines/olling-upgrade-centos7-alternator.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
```

Comment:
you are missing `r` in begining of the filename of this one

Reply:
Done